Change `prefer-query-selector` rule description to avoid confusion (#408)
sindresorhus/eslint-plugin-unicorn: docs/rules/prefer-query-selector.md

# Prefer `.querySelector()` over `.getElementById()`, `.querySelectorAll()` over `.getElementsByClassName()` and `.getElementsByTagName()`

> It's better to be use one method to query DOM elements.

Reviewer: I'm not a native speaker, but this sentence is not right to me

Author: My bad. Fixed it now.

> It's better to use a single method to query DOM elements.

Reviewer: ```suggestion
It's better to use the same method to query DOM elements.
```

Reviewer: Can you commit suggestion
